backoffice/views.py

```diff
@@ -305,3 +306,49 @@ def vehicles(request):
     context["page_range"] = page_range
 
     return render(request, "backoffice/vehicles.html", context)
+
+
+@require_backoffice_access
+def lock(request, id):
+    context = {
```

Comment:
We don't need a context anymore because the function returns a redirect rather than rendering a template.